docs/content/preview/develop/tutorials/AI/mcp-server.md

```diff
@@ -0,0 +1,190 @@
+---
+title: YugabyteDB MCP server
+headerTitle: MCP server
+linkTitle: MCP server
+description: Get LLMs to interact directly with YugabyteDB
+headcontent: Get LLMs to interact directly with YugabyteDB
+menu:
+  preview_develop:
+    identifier: tutorials-mcp-server
+    parent: tutorials-ai
+    weight: 20
+type: docs
+---
+
+This tutorial walks you through using the Model Context Protocol (MCP) to allow an AI application to access, query, analyze, and interpret data in your YugabyteDB database, using only natural language prompts.
+
+The tutorial uses a YugabyteDB cluster running the [Northwind dataset](../../../../sample-data/northwind/). You connect Claude to this database using MCP, then explore it using natural language prompts.
+
+## YugabyteDB MCP Server
+
```

Comment:
You can add about YB MCP server here or link to it - https://deploy-preview-28724--infallible-bardeen-164bc9.netlify.app/preview/explore/going-beyond-sql/gen-ai-apps/